scale tiflash: add a SQL to check tiflash table replica (#11244)
pingcap/docs: troubleshoot-lock-conflicts.md

    * Monitoring data through Grafana

>         On the `KV Errors` panel in the TiDB dashboard, there are two monitoring metrics `Lock Resolve OPS` and `KV Backoff OPS` which can be used to check read-write conflicts in the transactions. If the values of both `not_expired` and `resolve` under `Lock Resolve OPS` increase, there might be many read-write conflicts. The `not_expired` item means that the transaction's lock has not timed out. The `resolve` item means that the other transaction tries to clean up the locks. If the value of another `tikvLockFast` item under `KV Backoff OPS` increases, there might also be read-write conflicts.

Reviewer: ```suggestion
        On the `KV Errors` panel in the TiDB dashboard, `not_expired/resolve` under `Lock Resolve OPS` and `tikvLockFast` under `KV Backoff OPS` are two monitoring metrics that can be used to check read-write conflicts in transactions. If the values of both the two metrics increase, there might be many read-write conflicts. The `not_expired` item means that the transaction's lock has not timed out. The `resolve` item means that the other transaction tries to clean up the locks. The `tikvLockFast` item means that read-write conflicts occur.
```
